framework/src/main/java/org/checkerframework/common/util/count/AnnotationStatistics.java

```diff
@@ -66,6 +67,12 @@
 
     @Override
     public void typeProcessingOver() {
+        Log log = getCompilerLog();
+        if (log.nerrors != 0) {
+            System.out.printf(
+                    "Compilation issued %d error(s). Report might be partial.%n", log.nerrors);
```

Comment:
This is easy for a user to overlook.  I think it's better to produce no output than potentially incorrect output.  Same comment elsewhere.